collector/hwmon_linux.go

```diff
@@ -107,6 +107,9 @@ func sysReadFile(file string) ([]byte, error) {
 	if err != nil {
 		return nil, err
 	}
+	if n < 0 {
+		return nil, fmt.Errorf("failed to read file: %w", file)
```

Comment:
```
collector/hwmon_linux.go:112:15: fmt.Errorf format %w has arg file of wrong type string
```
I guess you meant to use `%q` maybe?

Reply:
Yes, typo. That's what I get for writing patches between meetings.